stakingModel/test/SimpleStaking.e2e.test.js

```diff
@@ -0,0 +1,66 @@
+const SimpleDAIStaking = artifacts.require("SimpleDAIStaking");
+const DAIMock = artifacts.require("DAIMock");
+const cDAIMock = artifacts.require("cDAIMock");
+
+const fse = require("fs-extra");
+
+const BN = web3.utils.BN;
+export const NULL_ADDRESS = "0x0000000000000000000000000000000000000000";
+const NETWORK = "test";
+
+contract("SimpleDAIStaking - `test` network e2e tests", ([founder, staker]) => {
+  let dai;
+  let cDAI;
+  let simpleStaking;
+
+  before(async function() {
+    const file = await fse.readFile("releases/deployment.json", "utf8");
+    const deployment = await JSON.parse(file);
+    const addresses = deployment[NETWORK];
+    const DAI_ADDRESS = addresses["DAI"];
+    const cDAI_ADDRESS = addresses["cDAI"];
+    const STAKING_ADDRESS = addresses["DAIStaking"];
+    dai = await DAIMock.at(DAI_ADDRESS);
+    cDAI = await cDAIMock.at(cDAI_ADDRESS);
+    simpleStaking = await SimpleDAIStaking.at(STAKING_ADDRESS);
+  });
+
+  it("should mint new dai", async () => {
```

Comment:
we dont test mocks only our contracts

Reply:
db43a7633912fed7aa89a7ff580fbe5238d650b5